flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -762,13 +763,10 @@ class Table(
     * @tparam T The data type that the [[TableSink]] expects.
     */
   def writeToSink[T](sink: TableSink[T]): Unit = {
-
-    def queryConfig = this.tableEnv match {
-      case s: StreamTableEnvironment => s.queryConfig
-      case b: BatchTableEnvironment => new BatchQueryConfig
-      case _ => null
+    val queryConfig = Option(this.tableEnv) match {
```

Comment:
`tableEnv` should never be `null`. Can we make this `def queryConfig = this.tableEnv.queryConfig`?

Reply:
ah, I see. Makes sense.